packages/cli/test/e2e/voluntaryExitRemoteSigner.test.ts

```diff
@@ -0,0 +1,171 @@
+import path from "node:path";
+import fs from "node:fs";
+import {GenericContainer, Wait, StartedTestContainer} from "testcontainers";
+import tmp from "tmp";
+import {retry} from "@lodestar/utils";
+import {ApiError, getClient} from "@lodestar/api";
+import {config} from "@lodestar/config/default";
+import {interopSecretKey} from "@lodestar/state-transition";
+import {spawnCliCommand, execCliCommand} from "@lodestar/test-utils";
+import {getMochaContext} from "@lodestar/test-utils/mocha";
+import {testFilesDir} from "../utils.js";
+
+let web3signerUrl: string;
+// using the latest image to be alerted in case there is a breaking change
+const web3signerVersion = "latest";
```

Comment:
As per discussions in https://github.com/ChainSafe/lodestar/issues/5942, we only want to use pinned versions to make test runs deterministic.

I'd suggest we just pin it to the latest version [23.11.0](https://github.com/Consensys/web3signer/releases/tag/23.11.0) and also update it in the other test case.

https://github.com/ChainSafe/lodestar/blob/8afbc98b561d714fdadd1946ec2a5aab83f479de/packages/validator/test/e2e/web3signer.test.ts#L20